### ISSUE TITLE
warmup_steps not used

### ISSUE THREAD

**Issue by robintibor** [y0ast/Glow-PyTorch#3]
It seems the `warmup_steps` parameter defined here:
https://github.com/y0ast/Glow-PyTorch/blob/7f14f9c8d47d5eb0ea0b7ef0f8a2c5d8f1117660/train.py#L327-L329

and in signature here:
https://github.com/y0ast/Glow-PyTorch/blob/7f14f9c8d47d5eb0ea0b7ef0f8a2c5d8f1117660/train.py#L78

is actually never used, only `warmup` parameter:
https://github.com/y0ast/Glow-PyTorch/blob/7f14f9c8d47d5eb0ea0b7ef0f8a2c5d8f1117660/train.py#L327-L329
 is used, here:
https://github.com/y0ast/Glow-PyTorch/blob/7f14f9c8d47d5eb0ea0b7ef0f8a2c5d8f1117660/train.py#L105

Help could also be a bit more informative in my view :) For example, like `Warmup learning rate linearly from zero to learning rate specified with --lr in this number of epochs`

Also, fresh parameter seems only used within the main-script-block but not in the main function, but not so important :D

**Comment by robintibor** [comment]
Ok cool! Just for clarification on my last question and for future reference: Now I understand that since you call `scheduler.step()` in `Events.EPOCH_COMPLETED` number of epochs refers, as expected, to epochs  going through whole dataset, not number of minibatches or something like that.
https://github.com/y0ast/Glow-PyTorch/blob/7f14f9c8d47d5eb0ea0b7ef0f8a2c5d8f1117660/train.py#L217-L221

**Comment by y0ast** [comment]
Yep that's correct, it's updated per epoch.
